### DISCUSSION TITLE
md.xyz file

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1467] [General]
Hello everyone,

I can see 7 columns in the md.xyz file but what does this 7 columns refer to? 
Which columns refer to the atomic positions of the atoms in the system?

**Comment by aradi** [comment]
1-3 cols: x,y,z coordinates of the atom, 4. col: population 5-7 cols: velocities in AA/ps.

**Reply by achyuttt** [reply]
What will be the units of x,y,z coordinates in md.xyz files? If my initial POSCAR file gives the units in Angstrom? as an input to the DFTB.
